### PR TITLE
New admin dashboard overview layout

### PR DESCRIPTION
**Pull request by lukeperrin10**
pt story: https://www.pivotaltracker.com/story/show/178228666 

## User Story 
```
As an Editor
To understand company performance
I need an overview of key performance indexes
```

## Suggested changes 

- New overview dashboard for Editor
- Includes statistics from Stripe and internal API
- Separated between 5 key value indicators and 2 graphs

## Dependencies

- react-responsive
- rechart

## Lessons learned 

- How to implement some responsiveness to webpage 
- Media Query 
- using rechart

## Screenshots
https://recordit.co/Ll59hM4bQK
image

### CODE REVIEW

**Review by DavDan1 — APPROVED** [review]

**Comment by DavDan1** [comment]
Looks crazy good!! Great job guys!

**Review by arunbhalli — APPROVED** [review]

File: src/components/StatsGraphs.jsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  CartesianGrid,
+  Legend,
+  AreaChart,
+  Area,
+} from 'recharts';
+
+const StatsGraphs = ({ data }) => {
+  const isSmallScreen = useMediaQuery({ query: '(max-width: 1400px)' });
+  const { articles_timeline, subscribers } = data;
+  const { error } = useSelector((state) => state);
```
Comment (offcast-dev) [inline src/components/StatsGraphs.jsx:20]: maybe error is not very descriptive, something like stripeError?

**Review by offcast-dev — APPROVED** [review]
small suggestion, not a blocker. Amazing job!

**Review by tochman — APPROVED** [review]
Good job

**Pull request merged**